pom.xml

```diff
@@ -164,7 +157,12 @@
             <version>6.11</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-core</artifactId>
+            <version>1.7.1</version>
```

Comment:
This is not the practise that we provide the check other places

Reply:
Changed